docs/scripts/gen_reference.py

```diff
@@ -144,6 +146,7 @@ def convert_crd_to_resource(crd: Dict, service) -> Resource:
     return Resource(
         name=crd['spec']['names']['kind'],
         service=service,
+        service_package=f"{service}-controller/apis" if service_package == "" else service_package,
```

Comment:
this seems backward to me, no?

Reply:
The Pythonic ternary operator just feels backwards to me in general. But this is correct. `{true_statement} if {condition} else {false_statement}`

Reply:
Hmm let me just rewrite this as a Python default argument value